resource-adapter/resource-adapter-deployment/README.adoc

```diff
@@ -0,0 +1,173 @@
+= Datasource via Deployment.
```

Comment:
Needs to be updated to the example, looks like a copy/paste from Datasource deployment example that wasn't updated